mirrord-agent/src/steal.rs

```diff
@@ -0,0 +1,374 @@
+use std::{
```

Comment:
Have this module be under a `tcp` module, so we'll end up having a `tcp::sniffer`, `tcp::steal`, and `tcp::outgoing` (or something similar).

With this, should probably move some of the more fundamental (and shared) types to the `tcp.rs` file itself.

Reply:
that's a good suggestion, but I think very soon we'll have the logic for UDP and it'll be part of the same module..?

Reply:
Makes sense, if they're going to be sharing the same `struct`s and functions then leaving as-is is fine. Maybe when we start working on udp and the differences begin to appear (if any) we can refactor as modules then.